src/services/api/AxiosInstance.ts

```diff
@@ -57,7 +60,12 @@ const requestFormatter = async (axiosConfig: AxiosRequestConfig) => {
       },
     },
   })
-  logger.info(`Github login user made call to Github API: ${axiosConfig.url}`)
+  const requestMethod = axiosConfig.method
+    ? axiosConfig.method
+    : "undefined method"
```

Comment:
Similarly as the previous comment, could we modify this to:
`const requestMethod = axiosConfig?.method ?? "undefined method";`

Reply:
simplified ternary operator to `??`